llvm/lib/Target/AIE/AIEMachineScheduler.h

```diff
@@ -37,6 +37,7 @@ std::vector<AIE::MachineBundle> computeAndFinalizeBundles(SchedBoundary &Zone);
 class AIEPostRASchedStrategy : public PostGenericScheduler {
   /// Maintain the state of interblock/loop-aware scheduling
   AIE::InterBlockScheduling InterBlock;
+  MutateInstructionMap MutateInstruction;
```

Comment:
As I understood, we use this map to store the information of an instruction B that should be allocated to a different slot to give a position to an instruction A (named new instruction). As we process one new instruction/time, we will use at least one position of this map. I think we can simplify this. What do you think?